index.html

```diff
@@ -1,23 +1,739 @@
----
-layout: default
----
+<!DOCTYPE html>
+<html lang="en">
 
-<div class="home">
+  <head>
 
-  <h1 class="page-heading">Posts</h1>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="">
+    <meta name="author" content="">
 
-  <ul class="post-list">
-    {% for post in site.posts %}
-      <li>
-        <span class="post-meta">{{ post.date | date: "%b %-d, %Y" }}</span>
+    <title>DashboardHub - Your Pipeline Heartbeat</title>
+    <link rel="shortcut icon" href="img/logos/favicon.ico">
```

Comment:
The assets repo has links to these files, so we dont have to include them but can use them on a CDN. 

For example `https://cdn.dashboardhub.io/favicon.ico`

Reply:
There is one image that I created(`dashboardhub_logo_whitebg.png`) that is not on the assets repo, so I've kept the relative path in for that particular image for now